docs/source/getting-started.md

```diff
@@ -0,0 +1,201 @@
+# Getting Started with ExecuTorch
+This section is intended to describe the necessary steps to take PyTorch model and run it using ExecuTorch. To use the framework, you will typically need to take the following steps:
+- Install the ExecuTorch python package and runtime libraries.
+- Export the PyTorch model for the target hardware configuration.
+- Run the model using the ExecuTorch runtime APIs on your development platform.
+- Deploy the model to the target platform using the ExecuTorch runtime.
+
+## Installation
+To use ExecuTorch, you will need to install both the Python package and the appropriate platform-specific runtime libraries.
+
+Pip is the recommended way to install the ExecuTorch python package. This package includes the dependencies needed to export a PyTorch model, as well as Python runtime bindings for model testing and evaluation. It is common to install the package within a Python virtual environment, in order to meet the Python and dependency version requirements.
+
+```
+pip install executorch
+```
+
+To build the framework from source, see [Building From Source](TODO).
+
+Backend delegates may require additional dependencies. See the appropriate backend documentation for more information.
+
+#### System Requirements
+The following are required to install the ExecuTorch host libraries, needed to export models and run from Python. Requirements for target end-user devices are backend dependent. See the appropriate backend documentation for more information.
```

Comment:
"run from Python" -> change to "run". 

The reason is that we're listing g++ requirements below, which is confusing.

Reply:
I thought that we needed a C++ compiler to install the pip package, as some parts were locally compiled at install time? I could be wrong on that, though.

@dbort Do you know if a C++ compiler is required to install the pip package?